Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -223,17 +228,15 @@ protected async Task OnTextKeyDownHandler( KeyboardEventArgs eventArgs )
 
             if ( IsConfirmKey( eventArgs ) && !string.IsNullOrEmpty( CurrentSearch ) )
             {
-                if ( IsMultiple )
+                if ( IsMultiple && FreeTyping && ActiveItemIndex < 0 )
```

Comment:
Are we sure `FreeTyping` check belongs here? Anyway I'll test the component later.